pkgs/hooks/lib/src/api/build_and_link.dart

```diff
@@ -11,7 +11,8 @@ import '../validation.dart';
 
 /// Builds assets in a `hook/build.dart`.
 ///
-/// Meant to be used in build hooks (`hook/build.dart`).
+/// If a build hook is defined (`hook/build.dart`) then `build` must be called
+/// by that hook, even if the [builder] function has no work to do.
```

Comment:
Ditto for `link`

Reply:
I don't see another comment...

Reply:
Search for "Meant to be used in link hooks (`hook/link.dart`)."

Reply:
Done. Could you check the phrasing.

Reply:
Theoretically, users can product the `output.json` file themselves. They could even decide not to use `package:hooks` at all, but they'd need to reimplement everything.

```
/// If a build hook is defined (`hook/build.dart`) then `build` must be called
/// by that hook to write the [BuildInput.outputFile], even if the [builder] function has no work to do.
```

Reply:
Thanks!